src/preferences/dialog/dlgprefmixer.cpp

```diff
@@ -717,6 +744,22 @@ void DlgPrefMixer::slotApply() {
     storeEqShelves();
 }
 
+void DlgPrefMixer::applyXFader() {
+    m_xfModeCO->set(m_xFaderMode);
+    m_xfCurveCO->set(m_xFaderCurve);
+    m_xfReverseCO->set(m_xFaderReverse ? 1.0 : 0.0);
+    m_xfCalibrationCO->set(m_xFaderCal);
+    if (m_xFaderReverse != (m_xfReverseCO->get() > 0)) {
+        double position = m_crossfader.get();
+        m_crossfader.set(0.0 - position);
```

Comment:
negative? Should've this been `1.0 - position`?

Reply:
`[Master],crossfader` range is -1..0..1, so 0 - position mirrors the value.
Or am I missing something?
(didn't change this btw)

But I made a mistake here, the CO is set and than compared to maybe flip the xfader, then set again.
Fixed.

Reply:
> range is -1..0..1, so 0 - position mirrors the value.
Or am I missing something?

Ah, no I assumed this set the parameter and thus was in the normalized $[0,1]$ range